File: FrontEnd/src/app/models/PedidoDto.ts
export interface PedidoDto {
  id: number;
  nomeCliente: string;
  emailCliente: string;
  pago: boolean;
  valorTotal: number;
  itensPedido: {
    id: number;
    idProduto: number;
    nomeProduto: string;
    valorUnitario: number;
    quantidade: number;
  }[];
}

File: BackEnd/Stefanini.Domain/Dto/PedidoDto.cs
namespace Stefanini.Domain.Dto
{
    public class PedidoDto
    {
        public int Id { get; set; }
        public string NomeCliente { get; set; }
        public string EmailCliente { get; set; }
        public bool Pago { get; set; }
        public decimal ValorTotal { get; set; }
        public List<ItensPedidoDto> ItensPedido { get; set; }
    }
}
